meta-resin-common/recipes-connectivity/libmbim/files/0001-avoid-iconv.patch

```diff
@@ -0,0 +1,143 @@
+From 6f87e17cd57bd735711349711ccf30b8746c2fcb Mon Sep 17 00:00:00 2001
+From: Aleksander Morgado <aleksander@aleksander.es>
+Date: Wed, 13 Jul 2016 11:36:11 +0200
+Subject: libmbim-glib,message: avoid iconv()
+
+There are systems out there that come with a fake iconv() implementation that
+don't support translations to/from UTF-16LE. We can easily avoid that by using
+the built-in translations to UTF-16HE provided by GLib itself, we just need to
+take care of converting from HE to/from LE.
+
```

Comment:
Hi, can you also add a "Signed-off-by" and "Upstream-Status" for this patch like in this patch? https://github.com/resin-os/meta-resin/blob/master/meta-resin-common/recipes-connectivity/dnsmasq/files/0001-Treat-REFUSED-not-SERVFAIL-as-an-unsuccessful-upstre.patch#L25

Only in your case the "Upstream-Status" would be "Backport" as I see that patch has made it into the libmbim git repo but they haven't released version 1.15 yet.
Thank you